core/storage/question/gae_models_test.py

```diff
@@ -523,6 +523,20 @@ def test_get_questions_with_no_skills(self):
             get_question_skill_links_equidistributed_by_skill(1, []))
         self.assertEqual(question_skill_links, [])
 
+    def test_get_questions_with_zero_count(self):
```

Comment:
Suggest either ``test_get_questions_with_zero_counts`` (pluralize) or split into two tests:
- ``test_get_questions_with_zero_question_count_returns_empty``
- ``test_get_questions_with_zero_skill_count_returns_empty``

(The latter is more in-line with the one-behavior-per-test principle).